src/components/Admin/adminTable.tsx

```diff
@@ -91,10 +91,14 @@ const AdminTable: React.FC = () => {
 
   return (
     <div className="px-4">
-      <div className="mb-8 mt-4">
-        <div className="flex items-center justify-between mb-4">
-          <div className="flex space-x-4">
-            <AddAdmin onAddAdmin={handleAddAdmin} />
+            <div className="mb-8 mt-4 ">
+              <div className="flex items-center justify-between mb-4 ">
+                <div className="flex-grow">
+                  <h2 className="text-xl font-semibold">Admin Table</h2>
+                </div>
+                <div>
+                  {" "}
```

Comment:
Don't use {" "} to add spacing to title, use css for this kind of things
update all instances in PR where this is used

Reply:
fix all the instances of this  {" "} approach